data/part-1/5-calculating.md

```diff
@@ -700,13 +700,13 @@ System.out.println(result);
 
 <sample-output>
 
-1
+1.0
 
 </sample-output>
 
 <!-- Nytkin tulostus on 1, sillä eka ja toka ovat (yhä) kokonaislukuja. -->
 
-The output 1 again, since first and second are (still) integers.
+The output is 1.0, since the variable `result` is of type `double` (***promotion*** `int` -> `double`).
```

Comment:
This terminology is not used elsewhere in the part, and may confuse rather than help explain the concept.